README.md

```diff
@@ -1,21 +1,70 @@
-# prismarine-template
-[![NPM version](https://img.shields.io/npm/v/prismarine-template.svg)](http://npmjs.com/package/prismarine-template)
-[![Build Status](https://github.com/PrismarineJS/prismarine-template/workflows/CI/badge.svg)](https://github.com/PrismarineJS/prismarine-template/actions?query=workflow%3A%22CI%22)
+# prismarine-realms
+[![NPM version](https://img.shields.io/npm/v/prismarine-realms.svg)](http://npmjs.com/package/prismarine-realms)
+[![Build Status](https://github.com/PrismarineJS/prismarine-realms/workflows/CI/badge.svg)](https://github.com/PrismarineJS/prismarine-realms/actions?query=workflow%3A%22CI%22)
 [![Discord](https://img.shields.io/badge/chat-on%20discord-brightgreen.svg)](https://discord.gg/GsEFRM8)
-[![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-template)
+[![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-realms)
 
-A template repository to make it easy to create new prismarine repo
+Node.JS Wrapper around both the Java & Bedrock Realms API
+
+## Installation
+```shell
+npm install prismarine-realms
+```
 
 ## Usage
 
+### RealmAPI
```

Comment:
I assume this is meant to be used eventually in mineflayer
if that's the case, could you add an example here how that would be used

Reply:
Yeah will be adding that once the implementation into the protocol libs is done and we've designed a way for users to choose which Realm